Gruntfile.js

```diff
@@ -234,6 +249,10 @@ module.exports = function (grunt) {
         },
         "node": {
             "version"       : "6.3.1"
+        },
+        "icu": {
+            "url"           : "http://s3.amazonaws.com/files.brackets.io/icu",
```

Comment:
ICU doesn't provide built versions for all its platforms. so I had to manually build it and then upload it on S3, this process is same as we do for CEF